hack/generate/samples/internal/ansible/molecule.go

```diff
@@ -0,0 +1,147 @@
+// Copyright 2020 The Operator-SDK Authors
```

Comment:
@fabianvf and @asmacdo 
Now, the Molecule mock data is gen and we update it as we need to. So, is no longer required the dirs:

<img width="536" alt="Screen Shot 2020-10-11 at 15 34 55" src="https://user-images.githubusercontent.com/7708031/95687027-04c95e00-0bf9-11eb-9f1a-4540438ef5dc.png">

And copy the full directories which not allow are we not let we testing all changes with the latest changes in the scaffold. See;  https://github.com/operator-framework/operator-sdk/blob/master/hack/tests/e2e-ansible-molecule.sh#L34-L50